src/http/routes/admin/tenants.ts

```diff
@@ -364,6 +352,7 @@ export default async function routes(fastify: FastifyInstance) {
     }
 
     await multitenantKnex('tenants').insert(tenantInfo).onConflict('id').merge()
+    await generateUrlSigningJwk(tenantId)
```

Comment:
should this be in the same transaction as the create tenant?

Reply:
Makes sense. Updated here and in POST.